GraphQL.md

```diff
@@ -44,22 +52,30 @@ public class Book {
     public Set<Author> authors;
 }
 ```
-
 ```java
 @Include(rootLevel = false)
 @Entity
 public class Author {
     @Id public long id;
     public String name;
     public Set<Book> books;
+    public Set<Publisher> publishers;
+}
+```
+```java
+@Include(rootLevel = false)
+@Entity
+public class Publisher {
+    @Id public long id;
+    public String name;
+    public Set<Book> books;
```

Comment:
We should be adding the @OneToOne @OneToMany annotations on all of these entities